environment.yml

```diff
@@ -54,6 +55,7 @@ dependencies:
   - requests >=1.2.3
   - selenium >=3.8
   - scipy
+  - pre-commit
```

Comment:
This should get added to the test envs in the CI directory too, I think? They won't be used in CI of course but I do refer people to those env files in order to make it easier to set up for a specific python version. cc @tcmetzger what do you think?

Reply:
Yes, I think it would make it easier for people referring to those files. As part of the dev docs update I'm working on, I will also point out that new dependencies should be added in `environment.yml` AND the CI files.

Reply:
I added it to all the `ci/environment-test-*.yml`, should I also add it to other `environment*.yml`?

Reply:
I don't think that's necessary.